app/Containers/AppSection/Authorization/UI/API/Routes/DetachPermissionsFromUser.v1.private.php

```diff
@@ -22,6 +22,6 @@
 use App\Containers\AppSection\Authorization\UI\API\Controllers\DetachPermissionsFromUserController;
 use Illuminate\Support\Facades\Route;
 
-Route::post('users/permissions/detach', [DetachPermissionsFromUserController::class, 'detachPermissionFromUser'])
+Route::post('users/{id}/permissions/detach', [DetachPermissionsFromUserController::class, 'detachPermissionFromUser'])
```

Comment:
I think this would be better as `Route::delete('users/{id}/permissions',...` to reflect the intent in the http verb